static/app/utils/replays/replayReader.tsx

```diff
@@ -162,6 +162,22 @@ export default class ReplayReader {
         finishedAtDelta: endTimestampMs - replayRecord.finished_at.getTime(),
       };
 
+      const metricData = {
+          unit: 'millisecond',
+          tags: {
+            // This is a boolean to reduce cardinality -- technically this can
+            // match 7.8.x, but replay wasn't released in that version, so this should be fine
+            recentSdkVersion: replayRecord.sdk.version.startsWith('7.8'),
+          }
+      };
+
+      if (this.timestampDeltas.startedAtDelta !== 0) {
+        Sentry.metrics.distribution('replay.start-time-delta', this.timestampDeltas.startedAtDelta, metricData);
+      }
+      if (this.timestampDeltas.finishedAtDelta !== 0) {
+        Sentry.metrics.distribution('replay.end-time-delta', this.timestampDeltas.finishedAtDelta, metricData);
+      }
```

Comment:
Can this go into `useLogReplayDataLoaded`? in order to trigger that hook the replay has to load successfully, and fully first. 

If the logging is in here i worry (but havn't fully checked) that we might get multiple logs emitted as large replays do their incremental loading.

I think mainly though, if there's an error we will still instantiate the ReplayReader class without errors/breadcrumb lists